consensus/spos/consensusMessageValidator_test.go

```diff
@@ -533,8 +533,15 @@ func TestCheckConsensusMessageValidity_ErrMessageTypeLimitReached(t *testing.T)
 		ChainID: chainID, MsgType: int64(bls.MtBlockBodyAndHeader),
 		Header: headerBytes, BlockHeaderHash: headerHash, PubKey: pubKey, Signature: sig, RoundIndex: 10,
 	}
-	err := cmv.CheckConsensusMessageValidity(cnsMsg, "")
-	assert.True(t, errors.Is(err, spos.ErrMessageTypeLimitReached))
+	t.Run("second time the message is received should still accept", func(t *testing.T) {
+		err := cmv.CheckConsensusMessageValidity(cnsMsg, "")
```

Comment:
should there be an cmv.AddMessageTypeToPublicKey(pubKey, 10, bls.MtBlockBodyAndHeader) here?